### ISSUE TITLE
WGI resources - Edit Only or New assets as well

### ISSUE THREAD

**Issue by dbrownukk** [dbrownukk/EFD_v2#128]
The whole generate spreadsheet, populate spreadsheet, parse spreadsheet results in the need for validation in the WGI resource tabs once the spreadsheet data is parsed. Validation 'infers' the checking of existing data. 

Since the resources are just being validated is there a need to Create new resource assets? 

If there is then it needs a little work to populate default values and LOVs for a new record and validate them.

**Comment by daiClegg** [comment]
Yes it is conceivable that the user may want to add further resources.
But i have tested the current implementation and it is acceptable, with no more than minor issues,

One i noted was that form Non-Food {Purchase i have to enter a value for the entered Resource sub-type (ie what would have been entered on the spreadsheet. 
Strictly that is unnecessary, and it isn't required to add an Employment.

So let's keep this enhancement open, but park it in the ice box.

**Comment by daiClegg** [comment]
Added to OIHM; the minor tweaks to UX to make adding new resources seamless are worth having.
This is, therefore, a Feature for OIHM & an Enhancement for OHEA
